### PR TITLE
fix(swap): use gas asset for ccm egress fee [WEB-1424]

### PR DESCRIPTION
**Pull request by GabrielBuragev**
The egress fees were stored using the wrong asset.

### CODE REVIEW

File: packages/swap/src/routes/swap.ts [hunk 1]
```diff
@@ -145,7 +145,7 @@ router.get(
       pendingDeposit?.transactionHash ??
       failedSwap?.depositTransactionRef ??
       undefined;
-
+    console.log(swapRequest?.fees, swap?.fees);
```
Comment (zoheb391) [inline packages/swap/src/routes/swap.ts:148]: missed

Comment (GabrielBuragev) [reply packages/swap/src/routes/swap.ts:148]: woops

**Review by zoheb391 — APPROVED** [review]
can we add a test that checks this

**Review by jerryafr — APPROVED** [review]

**Review by acdibble — APPROVED** [review]

File: packages/sdk/src/swap/assets.ts [hunk 2]
```diff
@@ -12,7 +12,7 @@ import type { AssetData } from './types';
 
 const isGasAsset = (asset: InternalAsset) => {
   const { chain } = assetConstants[asset];
-  return assetConstants[asset].asset === chainConstants[chain].gasAsset;
+  return asset === chainConstants[chain].gasAsset;
```
Comment (niklasnatter) [inline packages/sdk/src/swap/assets.ts:15]: im not sure if this is right - will this work for `ArbEth`?

Comment (niklasnatter) [reply packages/sdk/src/swap/assets.ts:15]: oh nevermind, just saw that you changed the type of `gasAsset`

**Pull request merged**